src/pages/ShoppingCart.jsx

```diff
@@ -74,13 +74,17 @@ class ShoppingCart extends Component {
   }
 
   render() {
+    const products = this.state;
     if (!localStorage.getItem('cart')) {
       return (
         <Fragment>
           <this.headerMount />
           <div className="empty-cart">
             <img src={empytCart} alt="carrinho vazio" />
             <span data-testid="shopping-cart-empty-message">Seu carrinho está vazio</span>
+            { products
+            ? <div className="items-in-cart" data-testid="shopping-cart-size">{products.length}</div>
+              : <div data-testid="shopping-cart-size" hidden></div>}
```

Comment:
aqui não tenho certeza se na sugestão que a indentação está 100% alinhada

```suggestion
            {
              products
               ? <div className="items-in-cart" data-testid="shopping-cart-size">{ products.length }</div>
               : <div data-testid="shopping-cart-size" hidden></div>
            }
```